### ISSUE TITLE
Canvas doesn't render in MAUI app on initial load

### ISSUE THREAD

**Issue by taublast** [taublast/DrawnUi.Maui#103]
### Discussed in https://github.com/taublast/DrawnUi.Maui/discussions/98

```xml
<views:Canvas Gestures="Enabled" HeightRequest="60" x:Name="Canvas" Grid.Row="1">
            <controls:ColorSlider HeightRequest="50" SelectedColor="{Binding SliderColor, Mode=OneWayToSource}"/>
</views:Canvas>
```

<sup>Originally posted by @nathenxbrewer October 24, 2024</sup>

> I'm trying to consume a SkiaSlider using the Canvas. For some reason, when the app loads, the Canvas is not rendered. If I change the xaml in any way, the app will Hot Reload and the canvas is then rendered. </div>

So here we have a situation that both Canvas and container layout inside have auto-size. The issue is that after measurement the measured layout size is not taken into consideration by the Canvas to adapt to measured content size.

 To be fixed.

**Comment by taublast** [comment]
@nathenxbrewer can you please share your xaml regarding the `Grid`? I see you have `Grid.Row="1"` so..
Or maybe you slider is different from the Sandbox one?

i tried to repro with the following, but no luck, canvas shows okay:

```xml
    <!--tried without definitions, same..-->
    <Grid RowDefinitions="80,*">


        <draw:Canvas
            x:Name="Canvas"
            Grid.Row="1"
            Gestures="Enabled"
            HeightRequest="60">
            <!--  one from the Sandbox project  -->
            <controls:SliderColor HeightRequest="50" />
        </draw:Canvas>

    </Grid>

```

**Comment by taublast** [comment]
i have a feeling this might fix it, to be verified when repro is available: c6687bdffb21b2949d2fc8b633aee311ba9d3222

**Comment by taublast** [comment]
Since we have no feedback would assume it was fixed, feel free to re-open if I was wrong.
